client/modules/IDE/actions/project.js

```diff
@@ -185,16 +185,15 @@ export function saveProject(selectedFile = null, autosave = false) {
       .then((response) => {
         dispatch(endSavingProject());
         const { hasChanges, synchedProject } = getSynchedProject(getState(), response.data);
+
+        dispatch(setNewProject(synchedProject));
+        dispatch(setUnsavedChanges(false));
+        browserHistory.push(`${mobile ? '/mobile' : ''}/${response.data.user.username}/sketches/${response.data.id}`);
```

Comment:
This isn't necessary anymore since the screen size determines if the mobile view is rendered, right? Which therefore means the `mobile` param isn't necessary anymore?

Reply:
Wow, I missed this completely. Thanks for catching that